### ISSUE TITLE
doc/core/howto/components.xhtml has 5 syntax errors in sample code

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3675]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ivank reported|
|-|-|
|Trac ID|trac#3675|
|Type|defect|
|Created|2009-02-26 19:55:26Z|

'as' is a keyword. doc/core/howto/components.xhtml uses 'as' as a variable name for "american socket".

Attachments:

* [components.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c47/c47ab2c905092f40e9ab88c4f01437566acb9fc2/2affd54468660bbc077335d655b81644153936f9.patch) (1143 bytes) - added by ivank on 2009-02-26 20:13:03Z - as -> am

<details><summary>Searchable metadata</summary>

```
trac-id__3675 3675
type__defect defect
reporter__ivank ivank
priority__normal normal
milestone__Python_2_6 Python-2.6
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1235678126000000 1235678126000000
changetime__1235682062000000 1235682062000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [26298]) Apply components.xhtml.patch

Author: ivank
Reviewer: exarkun
Fixes: [#3675](https://github.com/twisted/twisted/issues/3675)

In Python 2.6, "as" is a reserved word and may not be used as a variable.
Update the components howto examples so that they use a different variable
where "as" was being used.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
